storage/maria/unittest/CMakeLists.txt

```diff
@@ -134,3 +134,8 @@ ADD_EXECUTABLE(ma_pagecache_rwconsist2_1k-t ma_pagecache_rwconsist2.c)
 SET_TARGET_PROPERTIES(ma_pagecache_rwconsist2_1k-t PROPERTIES COMPILE_FLAGS "-DTEST_PAGE_SIZE=1024")
 MY_ADD_TEST(ma_pagecache_rwconsist2_1k)
 
+IF(CMAKE_SYSTEM_NAME MATCHES AIX)
+  FOREACH(target trnman-t ma_test_loghandler-t ma_test_loghandler_multigroup-t ma_test_loghandler_multithread-t ma_test_loghandler_pagecache-t ma_test_loghandler_long-t ma_test_loghandler_noflush-t ma_test_loghandler_first_lsn-t ma_test_loghandler_max_lsn-t ma_test_loghandler_purge-t ma_test_loghandler_readonly-t ma_test_loghandler_nologs-t ma_pagecache_single_1k-t ma_pagecache_single_8k-t ma_pagecache_single_64k-t ma_pagecache_consist_1k-t ma_pagecache_consist_64k-t ma_pagecache_consist_1kHC-t ma_pagecache_consist_64kHC-t ma_pagecache_consist_1kRD-t ma_pagecache_consist_64kRD-t ma_pagecache_consist_1kWR-t ma_pagecache_consist_64kWR-t ma_pagecache_rwconsist_1k-t ma_pagecache_rwconsist2_1k-t)
+    TARGET_LINK_OPTIONS(${target}  PRIVATE "-Wl,-berok")
```

Comment:
this is covered by MY_ADD_TESTS already, you do not need the berok  more that a single time.

Reply:
Done.